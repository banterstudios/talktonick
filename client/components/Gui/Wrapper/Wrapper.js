/* eslint-disable no-unused-vars */
import React from 'react'
/* eslint-enable no-unused-vars */

import glamorous from 'glamorous'

const Wrapper = glamorous.section({
  position: 'relative',
  width: '100%'
})

export default Wrapper
