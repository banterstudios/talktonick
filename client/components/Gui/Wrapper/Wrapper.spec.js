import React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Wrapper from './Wrapper'

describe('components/Gui/Wrapper', () => {
  describe('@renders', () => {
    it('in default state', () => {
      const component = mount(<Wrapper />)

      expect(toJson(component)).toMatchSnapshot()
      component.unmount()
    })
  })
})
