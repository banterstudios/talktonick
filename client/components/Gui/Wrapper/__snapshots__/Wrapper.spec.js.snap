// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Gui/Wrapper @renders in default state 1`] = `
<glamorous(section)>
  <section
    className="css-4kgshc"
  />
</glamorous(section)>
`;
